nilearn/tests/test_signal.py

```diff
@@ -257,12 +257,6 @@ def test_standardize():
 
     # transpose array to fit _standardize input.
     # Without trend removal
-    b = nisignal._standardize(a, standardize='zscore')
-    stds = np.std(b)
-    np.testing.assert_almost_equal(stds, np.ones(n_features))
-    np.testing.assert_almost_equal(b.sum(axis=0), np.zeros(n_features))
-
```

Comment:
Why are you removing this ?

Reply:
@bthirion it's just repeating exactly another test but with a deprecated value for `standardize`. The deprecated value is still tested here https://github.com/nilearn/nilearn/blob/01cfa3d531efb749e53abe0c57c5adf0bb844ee0/nilearn/tests/test_signal.py#L255-L256 so at least we know nothing is breaking but I don't see a reason to keep testing it with different combinations of parameters